shame_script.py

```diff
@@ -151,8 +161,9 @@ async def fetch_and_send_tasks() -> None:
                 )
 
                 message_content.append(
-                    f"*Tasks for {discord_user.mention}*\n```\n{table}\n```"
+                    f"*Tasks for {discord_user.mention}* | Streak: {user_score.streak}\n```\n{table}\n```"
                 )
+            database_session.commit()
```

Comment:
```suggestion
```
`.commit` is run as part of `with` context exit

Reply:
do you have a source for this?

empirically testing this is not the case

Reply:
I was wrong